django_spanner/utils.py

```diff
@@ -18,3 +19,17 @@ def check_django_compatability():
                 A=django.VERSION[0], B=django.VERSION[1], C=__version__
             )
         )
+
+
+def ensure_where_clause(sql):
```

Comment:
```suggestion
def add_dummy_where(sql):
```

Pretty confusing that this function name shows up twice, and with two different meanings of "ensure"!